fw/Descriptors.c

```diff
@@ -302,6 +212,45 @@ const USB_Descriptor_String_t PROGMEM ManufacturerString = USB_STRING_DESCRIPTOR
 // listed in the Device Descriptor.
 const USB_Descriptor_String_t PROGMEM ProductString = USB_STRING_DESCRIPTOR(L"BEEF BOARD");
 
+#define LedStringCount 17
+const USB_Descriptor_String_t PROGMEM led_name1 = USB_STRING_DESCRIPTOR(L"Button 1");
+const USB_Descriptor_String_t PROGMEM led_name2 = USB_STRING_DESCRIPTOR(L"Button 2");
+const USB_Descriptor_String_t PROGMEM led_name3 = USB_STRING_DESCRIPTOR(L"Button 3");
+const USB_Descriptor_String_t PROGMEM led_name4 = USB_STRING_DESCRIPTOR(L"Button 4");
+const USB_Descriptor_String_t PROGMEM led_name5 = USB_STRING_DESCRIPTOR(L"Button 5");
+const USB_Descriptor_String_t PROGMEM led_name6 = USB_STRING_DESCRIPTOR(L"Button 6");
+const USB_Descriptor_String_t PROGMEM led_name7 = USB_STRING_DESCRIPTOR(L"Button 7");
+const USB_Descriptor_String_t PROGMEM led_name8 = USB_STRING_DESCRIPTOR(L"E1");
+const USB_Descriptor_String_t PROGMEM led_name9 = USB_STRING_DESCRIPTOR(L"E2");
+const USB_Descriptor_String_t PROGMEM led_name10 = USB_STRING_DESCRIPTOR(L"E3");
+const USB_Descriptor_String_t PROGMEM led_name11 = USB_STRING_DESCRIPTOR(L"E4");
```

Comment:
I think we should use generic `Button 8/9/10/11` instead of `E1/2/3/4`. It would be strange for popn users to see E*, and it would also be consistent with the PCB silkscreen.

Reply:
Does pop'n work OOTB with stock firmware? I'd think it might require some modifications due to HID button layouts being different between it and IIDX.

Reply:
Indeed it does. @supervaka is passing 50s on it.